README.md

```diff
@@ -19,6 +19,12 @@ You must protect user privacy at all times, even when using prototypes. Prototyp
 - [Installation guide for new users (non technical)](https://govuk-prototype-kit.herokuapp.com/docs/install/introduction)
 - [Installation guide for developers (technical)](https://govuk-prototype-kit.herokuapp.com/docs/install/developer-install-instructions)
 
+## Node version requirements
+
+We always recommend you use the [current long term support (LTS) version of Node.js](https://github.com/nodejs/release#release-schedule).
```

Comment:
`We recommend you use the [current long term support (LTS) version of Node.js](https://github.com/nodejs/release#release-schedule).`